Mathlib/LinearAlgebra/TensorProduct/Basic.lean

```diff
@@ -13,22 +13,25 @@ import Mathlib.Tactic.SuppressCompilation
 /-!
 # Tensor product of modules over commutative semirings.
 
-This file constructs the tensor product of modules over commutative semirings. Given a semiring
-`R` and modules over it `M` and `N`, the standard construction of the tensor product is
-`TensorProduct R M N`. It is also a module over `R`.
+This file constructs the tensor product of modules over commutative semirings. Given a semiring `R`
+and modules over it `M` and `N`, the standard construction of the tensor product is `TensorProduct R
+M N`. It is also a module over `R`.
```

Comment:
please no line break inside a code block

Reply:
Roger. Is there anything I need to do now?

Reply:
I fixed the line break issue, as requested.